glean-core/src/database/mod.rs

```diff
@@ -68,11 +69,16 @@ impl Database {
 
     /// Iterates with the provided transaction function on the data from
     /// the given storage.
-    pub fn iter_store_from<F>(&self, lifetime: Lifetime, storage_name: &str, mut transaction_fn: F)
-    where
+    pub fn iter_store_from<F>(
+        &self,
+        lifetime: Lifetime,
+        storage_name: &str,
+        metric_key: Option<&str>,
+        mut transaction_fn: F,
```

Comment:
We're getting lots of arguments here.
Can we document what they mean?
Right now I know what `metric_key` is. I won't know when I look at this again on Monday.